packages/server/lib/modes/run.coffee

```diff
@@ -767,6 +767,7 @@ module.exports = {
       osName: sys.osName,
       osVersion: sys.osVersion,
       cypressVersion: pkg.version,
+      runUrl: runUrl,
```

Comment:
Super minor, not worth fixing (will merge), but you can omit values when they match the key names... now that I think about this why isn't this a lint rule... oh wait we're still on coffeescript, oh nevermind then.